oak-wrapped-guava/dependency-reduced-pom.xml

```diff
@@ -0,0 +1,90 @@
+<?xml version="1.0" encoding="UTF-8"?>

```

Comment:
Needs a license header.

Reply:
That was autogenerated and I believe not needed by us. Reconfigured the plugin not to generate it.